actors/market/src/types.rs

```diff
@@ -97,16 +96,13 @@ pub struct ActivateDealsParams {
 
 #[derive(Serialize_tuple, Deserialize_tuple)]
 pub struct ActivateDealsResult {
-    pub weights: DealWeights,
+    pub spaces: DealSpaces,
 }
 
 #[derive(Serialize_tuple, Deserialize_tuple, Debug, Clone, Default)]
-pub struct DealWeights {
+pub struct DealSpaces {
     pub deal_space: u64,
-    #[serde(with = "bigint_ser")]
-    pub deal_weight: DealWeight,
-    #[serde(with = "bigint_ser")]
-    pub verified_deal_weight: DealWeight,
+    pub verified_deal_space: u64,
```

Comment:
This u64 tops out at 18 EiB. While we'd have time to migrate it, I think this one is close enough to just use a BigInt from the beginning. Probably just BigInt, but we might need a new type alias later.